src/components/PurchaseOrder/POInvoices/POInvoicesContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Accordion } from '@folio/stripes/components';
+
+import {
+  INVOICES,
+  ORDER_INVOICES,
+} from '../../Utils/resources';
+
+import POInvoices from './POInvoices';
+
+class POInvoicesContainer extends Component {
+  static manifest = Object.freeze({
+    orderInvoicesRelns: ORDER_INVOICES,
+    invoices: INVOICES,
+  });
+
+  static propTypes = {
+    orderId: PropTypes.string.isRequired,
+    label: PropTypes.object.isRequired,
+    accordionId: PropTypes.string.isRequired,
+    mutator: PropTypes.shape({
+      orderInvoicesRelns: PropTypes.object.isRequired,
+      invoices: PropTypes.object.isRequired,
+    }).isRequired,
+    resources: PropTypes.object.isRequired,
+    vendors: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  static defaultProps = {
+    vendors: [],
+  };
+
+  componentDidMount() {
+    this.fetchOrderInvoices();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.orderId !== prevProps.orderId) {
```

Comment:
we have the same situation here, as with ApprovedBy user in ui-invoice. Can't stripes-connect resource handle a prop change?